usagov/usagov-2021
# usagov-1276-gray-footer-color: added css for footer color

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1276

## Description

Verify the gray desktop footer has changed from #DFE1E2 to #F0F0F0. In mobile, verify the gray footer which contains the email updates section has changed from #DFE1E2 to #F0F0F0
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto any page like http://localhost/
- Goto the footer section and verify the gray footer background color is #f0f0f0
- View the same page in mobile view and expand the footer. Verify the background color is #f0f0f0
- Also verify the email section in mobile has a background color #f0f0f0

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
